server/routes/induction/update/index.ts

```diff
@@ -140,6 +143,11 @@ export default (router: Router, services: Services) => {
   }
 
   if (config.featureToggles.induction.update.prePrisonEducationSectionEnabled) {
+    router.get('/prisoners/:prisonNumber/induction/qualifications', [
```

Comment:
I've only defined the GET route and controller handler method at this point, to prove that it works, specifically getting the Curious functional skills data as a request handler function